modules/checking-deployment-yaml-files.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * cli/checking-policy-compliance.adoc
+:_content-type: PROCEDURE
+[id="checking-deployment-yaml-files_{context}"]
+= Checking deployment YAML files
+
+.Procedure
+
+* Run the following command to check the build-time and deploy-time violations of your security policies in YAML deployment files:
+//TODO: Add link to security policies section
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl -e "$ROX_CENTRAL_ADDRESS" deployment check --file=_<yaml_filename>_
+----
++
+This command validates the following items:
```

Comment:
For clarity, the section in `checking-image-scan-result` w.r.t the required permissions as well as the `--force` flag, so maybe this should be extracted into a separate module?

So, essentially you can always force to re-scan images, and for that you need correct permissions.